dev-libs/processor-trace/processor-trace-1.6.1.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="Intel(R) Processor Trace decoder library"
+HOMEPAGE="https://github.com/01org/processor-trace"
+SRC_URI="https://github.com/01org/processor-trace/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT=0
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86 ~ppc-aix ~amd64-fbsd ~x86-fbsd ~amd64-linux ~arm-linux ~x86-linux ~x64-macos ~x86-macos ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
```

Comment:
These `KEYWORDS` are very optimistic for a library that does low-level parsing of x86-specific binary format. I suggest starting from `~amd64 ~x86` if you have tested it.

Reply:
> These KEYWORDS are very optimistic for a library that does low-level parsing of x86-specific binary format. I suggest starting from ~amd64 ~x86 if you have tested it.
It works only for amd64 (because of only modern intel processors). Without it CI reported «broken» (or I just misunderstood). Ok, I'll just remove it.

Reply:
You might also need to mask this use flag on all arches except `amd64` via `profiles/` directory.
Usually it means putting `sys-devel/gdb intel-pt` into `profiles/packages.use.mask` and unmasking it in arch profiles by putting `sys-devel/gdb -intel-pt` into `profiles/arch/amd64/package.use.mask`. (see various `jit` flag examples).

Are you sure it does not work for `x86`?

Reply:
Thx for the explanation. I'll fix it.

> Are you sure it does not work for x86?

It can be work on x86 **on** amd64 processor, but there is no support for x86 processors.

Reply:
fixed